### PR TITLE
Add userfunction for links to related websites

### PR DESCRIPTION
**Pull request by tobiaskresse**

### CODE REVIEW

File: Classes/Controller/EntityFactController.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function showAction()
             $viewArray = $apiAnswerDecode;
         }
 
-        $viewArray['describedBy'] = $apiAnswerDecode['describedBy'];
+        $viewArray["describedBy"] = $apiAnswerDecode["describedBy"];
```
Comment (albig) [inline Classes/Controller/EntityFactController.php]: Any reason for this change?

Please always use single quotes for array keys and strings. Double quotes are only used to add the newline character `\n` to a string.

See: https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.html#strings

File: Classes/Backend/ItemsProcFunc.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    
+    //constructing the wanted array
+    foreach ($apiAnswerDecode["sameAs"] as $item) 
+    {
+      $newItems[] = 
+      [$item['collection']['name'],$item['collection']['abbr']];
+    }
+    $config['items'] = $newItems;
+  }
+}
```
Comment (albig) [inline Classes/Backend/ItemsProcFunc.php]: Please configure your IDE to 

- always add a newline to the end of a file
- remove trailing spaces

You may configure your IDE to respect the .editorconfig file provided by TYPO3. Have a look at https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/Introduction.html

File: Classes/Backend/ItemsProcFunc.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  * populating the option for links
+  *
+  * @param array &$config configuration array
+  */
+  public function user_sameAsSelected(array &$config)
+  {
+    $search = $config['row']['settings.entityfacts.personality'];
+
+    //api commmunication and decoding
+    $apiAnswer = file_get_contents('http://hub.culturegraph.org/entityfacts/'.$search);
```
Comment (albig) [inline Classes/Backend/ItemsProcFunc.php]: We willl keep this in this pull request. In a next step, you should make this URL configurable. E.g. by TypoScript and/or Flexform.

Always make spaces before and after the string concanating operator `'.'`. See https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.html#strings

**Review by albig — CHANGES_REQUESTED** [review]
Only some tiny formatting whishes from my side.

Please fix this in a new commit to the repo.

File: Classes/Controller/EntityFactController.php (hunk 1 above)
Comment (tobiaskresse) [reply Classes/Controller/EntityFactController.php]: Once got a php error that wanted that change. I can't replicate the error now, so I'll happily change it back.

File: Classes/Backend/ItemsProcFunc.php (hunk 3 above)
Comment (albig) [inline Classes/Backend/ItemsProcFunc.php]:
```suggestion
    $apiAnswer = file_get_contents('http://hub.culturegraph.org/entityfacts/' . $search);
```

File: Classes/Backend/ItemsProcFunc.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    $apiAnswerDecode = json_decode ($apiAnswer, true);
+
+    
+    //constructing the wanted array
+    foreach ($apiAnswerDecode["sameAs"] as $item)
```
Comment (albig) [inline Classes/Backend/ItemsProcFunc.php]:
```suggestion
    foreach ($apiAnswerDecode['sameAs'] as $item)
```

**Review by albig — CHANGES_REQUESTED** [review]
Please accept the two suggestions.

**Review by albig — APPROVED** [review]

**Pull request merged**